### DISCUSSION TITLE
Iphone or iPad feature.

### DISCUSSION THREAD

**Discussion by WzrdManx90x** [ntsc-rs/ntsc-rs#182] [Ideas]
Is there like, anyway to add a ipa version of NTSC-RS?

**Comment by ghost** [comment]
Probably not soon.

**Comment by sakhalif456** [comment]
not i think,,,

**Comment by valadaptive** [comment]
I've added this to [the FAQ](https://ntsc.rs/docs/faq/#more-platforms):

> ## Can I use ntsc-rs on Android, iOS, or the Web?
> 
> Unfortunately, none of those platforms are supported at the moment. It's unlikely that apps for iOS or Android will ever be released, but in the long term, it's possible that I'll make an ntsc-rs webapp.
> 
> Mobile platforms are a chore to develop for:
> 
> -    On the Android side, Google is notorious for deprecating their libraries every couple of years, forcing you onto the treadmill of rewriting your software over and over.
> -    On the iOS side, Apple makes you fork over $99/yr for the privilege of being allowed on the App Store. I don't have any iOS hardware to test on, either.
